src/appleseed-max-impl/appleseedrenderer/resource.h

```diff
@@ -43,6 +42,7 @@
 #define IDC_SPINNER_ADAPTIVE_NOISE_THRESHOLD            232
 #define IDC_TEXT_NOISE_SEED                             233
 #define IDC_SPINNER_NOISE_SEED                          234
+#define IDC_CHECK_ENABLE_NOISE_SEED                     235
```

Comment:
Why are some constants prefixed with `RENDERERPARAMS` while others not?

Reply:
RENDERPARAMS prefixed ones refer to the ui-element resources which have to be defined as key-value pairs for combo-box selection menus in appleseedrenderer.cpp.

Reply:
Ok.